### DISCUSSION TITLE
Will support for YoloV10 be added?

### DISCUSSION THREAD

**Discussion by apalkk** [obss/sahi#1084] [Q&A] [answered]
YoloX support which is currently present is useful, but it would be great to have YoloV10

**Comment by fcakyon** [comment, marked as answer]
It should work with ultralytics yolov10 using:

```python
AutoDetectionModel.from_pretrained(
    model_type='yolov8',
    ...
)
```

You can try it in https://github.com/obss/sahi/blob/main/demo/inference_for_yolov8.ipynb
